Sts1CobcSw/Hal/Spi.cpp

```diff
@@ -2,9 +2,21 @@
 
 #include <rodos_no_using_namespace.h>
 
-
 namespace sts1cobcsw::hal
 {
+auto Initialize(Spi * spiClass, std::int32_t baudRate) -> void
```

Comment:
Unfortunately, `baudRate` must be of type `std::uint32_t` because `spi.init()` requires that.